api/schema/application_azure_resources_resource.go

```diff
@@ -27,12 +27,22 @@ var (
 						},
 					},
 					attributeWithMode: attributeWithMode{
-						Mode: ImmutableAttributeMode,
+						Mode: KeyAttributeMode,
+					},
+				},
+				"application_resource_ids": ListResourceAttributeWithMode{
+					ListAttribute: resource_schema.ListAttribute{
+						ElementType: types.StringType,
+						Description: "CloudSecure IDs of the resources in the CloudSecure application.",
+						Computed:    true,
+					},
+					attributeWithMode: attributeWithMode{
+						Mode: KeyAttributeMode,
```

Comment:
This can't be in the create request.
```suggestion
						Mode: IDAttributeMode,
```